java/solomiia_tymoshchuk/src/com/zoolatech/lecture4/tasks/_3/StringModifier.java

```diff
@@ -0,0 +1,6 @@
+package com.zoolatech.lecture4.tasks._3;
+
+@FunctionalInterface
+interface StringModifier {
+    abstract String modify(String initialString);
```

Comment:
all interface methods are `abstract` by default, you don't need to add this modifier